.scalafmt.conf

```diff
@@ -0,0 +1,2 @@
+version = "3.7.4"
+runner.dialect = scala3
```

Comment:
I think we need `scala213source3` here, otherwise it might fail on formatting some 2.13 code, I had such issues previously

Reply:
Or I can remove it completely. Current sources are formatted with Intellij (I guess). This file is only a side effect of me using metals which creates scalafmt configuration whenever one formats the document. But `scala213source3` works too!